src/Reveal.Sdk.Dom.Tests/Visualizations/GridVisualizationFixture.cs

```diff
@@ -0,0 +1,358 @@
+                          "FieldName": "Conversions",
+                          "Sorting": "None"
+                        },
+                        {
+                          "_type": "TabularColumnSpecType",
+                          "FieldName": "Spend",
+                          "Sorting": "None"
+                        }
+                      ]
+                    }
+                  }
+                ]
+                """;
+            var document = new RdashDocument("My Dashboard");
+
+            var excelDataSourceItem = new RestDataSourceItem("Marketing Sheet")
+            {
+                Subtitle = "Excel Data Source Item",
+                Url = "http://dl.infragistics.com/reportplus/reveal/samples/Samples.xlsx",
+                IsAnonymous = true,
+                Fields = new List<IField>
+                {
+                    new DateField("Date"),
+                    new NumberField("Spend"),
+                    new NumberField("Budget"),
+                    new NumberField("CTR"),
+                    new NumberField("Avg. CPC"),
+                    new NumberField("Traffic"),
+                    new NumberField("Paid Traffic"),
+                    new NumberField("Other Traffic"),
+                    new NumberField("Conversions"),
+                    new TextField("Territory"),
+                    new TextField("CampaignID"),
+                    new NumberField("New Seats"),
+                    new NumberField("Paid %"),
+                    new NumberField("Organic %")
+                }
+            };
+            excelDataSourceItem.UseExcel("Marketing");
+
+            document.Visualizations.Add(new GridVisualization("Grid", excelDataSourceItem)
+            {
+                IsTitleVisible = true,
+                Description = "Create Grid Visualization"
+            }
+            .SetColumns("Territory", "Conversions", "Spend")
+            .ConfigureSettings(settings =>
+            {
+                settings.IsPagingEnabled = true;
+                settings.IsFirstColumnFixed = true;
+                settings.FontSize = FontSize.Large;
+                settings.DateFieldAlignment = Alignment.Center;
+                settings.TextFieldAlignment = Alignment.Center;
+            }));
+
+            document.Filters.Add(new DashboardDataFilter("Spend", excelDataSourceItem));
+            document.Filters.Add(new DashboardDateFilter("My Date Filter"));
+
+            // Act
+            RdashSerializer.SerializeObject(document);
+            var json = document.ToJsonString();
+            var jObject = JObject.Parse(json);
+            var actualJArray = (JArray)jObject["Widgets"];
+            var expectedJArray = JArray.Parse(expectedJson);
+
+            // Assert
+            Assert.Equal(expectedJArray.Count, actualJArray.Count);
+
+            for (int i = 0; i < expectedJArray.Count; i++)
+            {
+                Assert.Equal(expectedJArray[i], actualJArray[i]);
```

Comment:
Same here, recheck, this is not correct. Pls make sure the asserts are meaningful